contracts/DxMgnPool.sol

```diff
@@ -114,6 +114,9 @@ contract DxMgnPool is Ownable {
             depositToken.approve(address(dx), depositAmount);
             dx.deposit(address(depositToken), depositAmount);
         }
+        if (isDepositTokenTurn()) {
```

Comment:
nit: no need to multiply with 2

Also maybe we can combine it with the if clause above?